### ISSUE TITLE
Allow to return custom RADIUS attributes inside the switch configuration itself

### ISSUE THREAD

**Issue by julsemaan** [inverse-inc/packetfence#5905]
**Is your feature request related to a problem? Please describe.**
Sometimes you want to return one or more custom RADIUS attributes that have values based on the role being returned and the switch you're configuring. Right now, you'd have to create a RADIUS filter for each role+location which generates a lot of filters (i.e. you have 10 roles and 10 location, you get 100 filters). You also can't use the device templates since the value depends on the role and dynamic values can only be vlan, roles, URLs, ACLs.

**Describe the solution you'd like**
In the switch configuration, add a "Role by RADIUS attributes" section (name can be different) and be able to define RADIUS attributes to return for each role

See screenshot:
![image](https://user-images.githubusercontent.com/3857942/95496246-86598b80-096e-11eb-8c3f-e41f47b39f63.png)

**Describe alternatives you've considered**
Ask @lzammit to create all the RADIUS filters. He said he's too busy running http://www.zammitcorp.com/
Use the switch templates but they don't have a section for each role.

**Comment by ZdenekPala** [comment]
I believe this FR is very beneficial.
Just to be sure, the "Role by RADIUS attributes" can be combined with "Role by VLAN ID" and "Role by Switch Role".
the final radius Access-Accept should contain the combined information.

**Comment by julsemaan** [comment]
> I believe this FR is very beneficial.
> Just to be sure, the "Role by RADIUS attributes" can be combined with "Role by VLAN ID" and "Role by Switch Role".
> the final radius Access-Accept should contain the combined information.

Yes that is correct, it will combine with any other one that is enabled

**Comment by PureOxygen-Dev** [comment]
Team, can I add my hand up also for this Feature Enhancement if it is been considered for 12.2 :-)
